recipes/ppqsort/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conan import ConanFile
+from conan.tools.layout import basic_layout
+from conan.tools.files import get, copy, replace_in_file
+from conan.tools.build import check_min_cppstd
+
+import os
+
+required_conan_version = ">=2.9.0"
+
+class PPQSortConan(ConanFile):
+    name = "ppqsort"
+    package_type = "header-library"
+
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/GabTux/PPQSort"
+    description = "Parallel Pattern Quicksort"
+    topics = ("algorithms", "sorting", "parallel", "header-only")
+
+    implements = ["auto_header_only"]
+    settings = "os", "arch", "compiler"
+    languages = "C++"
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def validate(self):
+        check_min_cppstd(self, 20)
+
+    def layout(self):
+        basic_layout(self)
+
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(
+            self.package_folder, "licenses"), src=self.source_folder)
+        copy(self, "include/*", self.source_folder, self.package_folder)
+
+    def package_info(self):
+        self.cpp_info.bindirs = []
+        self.cpp_info.libdirs = []
+
+        self.cpp_info.set_property("cmake_target_name", "PPQSort::PPQSort")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_file_name", "PPQSort")
        self.cpp_info.set_property("cmake_target_name", "PPQSort::PPQSort")
```

As the example in the upstream's readme shows:

```
CPMAddPackage(
NAME PPQSort
...
```